src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,26 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type SingleTodo = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<SingleTodo> = ({ todo }) => (
+  <>
+    <article className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': todo.completed,
+      },
+    )}
+    >
+      <h2 className="TodoInfo__title">{`${todo.title}`}</h2>
+      {todo.user && (
+        <UserInfo key={todo.userId} user={todo.user} />
+      )}
```

Comment:
you don't `.map` here, so no need for an explicit `key`
```suggestion
      {todo.user && (
        <UserInfo user={todo.user} />
      )}
```